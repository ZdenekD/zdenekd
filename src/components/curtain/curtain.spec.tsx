import {shallow} from 'enzyme';
import findComponent from '@/__test__/utils/helpers';
import Curtain from '.';

describe('Curtain', () => {
    it('renders without error', () => {
        const wrapper = shallow(<Curtain />);
        const component = findComponent(wrapper, 'component-curtain');

        expect(component.exists()).toBe(true);
    });
});
