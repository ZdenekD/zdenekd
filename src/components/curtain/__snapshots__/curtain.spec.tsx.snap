// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Curtain match snapshot 1`] = `
<div class="curtain"
     data-test="component-curtain"
     style="background:var(--curtain-base-color);transform:none"
>
</div>
`;
